config/locales/en.yml

```diff
@@ -444,98 +445,7 @@ en:
   #############################
   errors:
     arms:
-      manage: "Manage Arms"
```

Comment:
Note: A bunch of these YML segments were incorrectly placed here. I think this was the result of a bad merge somewhere beforehand. It was causing issues with this story so I fixed it.